src/main/java/sirius/biz/scripting/ScriptableEvents.java

```diff
@@ -53,11 +54,10 @@ public void handleEvent(ScriptableEvent event) {
      *
      * @param name the name of the dispatcher to fetch
      * @return the dispatcher for the current tenant with the given name or a NOOP dispatcher if no such dispatcher
-     * exists. Note, if an empty <tt>name</tt> is given, the first available dispatcher for the current tenant is used.
-     * This way, if exactly one dispatcher is present, it will be used in all import processes etc.
+     * exists.
      */
-    public ScriptableEventDispatcher fetchDispatcherForCurrentTenant(@Nullable String name) {
-        if (dispatcherRepository == null) {
+    public ScriptableEventDispatcher fetchDispatcherForCurrentTenant(String name) {
+        if (dispatcherRepository == null || Strings.isEmpty(name)) {
```

Comment:
I think this breaks the "dont show parameter for single script" approach

Reply:
I took a different approach hier: the parameter uses the `.hideWhen` method and hides itself when only one entry exists. This way we can differentiate if a single param exists and must be used OR if scripting has been disabled completely.

Reply:
Understood - perfect +1